src/Contract/Config/DeptracConfig.php

```diff
@@ -114,6 +123,8 @@ public function toArray(): array
             $config['paths'] = $this->paths;
         }
 
+        $config['analyser']['skip_deprecated'] = $this->skipDeprecated;
```

Comment:
This would not even work when an `analyser` would be present as it would get overridden by the analyser serialized to array.